CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ project(hiredis-cluster
   LANGUAGES C)
 
 set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_EXTENSIONS OFF)
```

Comment:
Add a comment before these two lines? Idea:

Use plain C99 (-std=c99) without extensions like GNU C (-std=gnu99)

Reply:
Good idea, will add this before merging